<launch>
    <arg name="world_frame_name"    default="world"/>
    <arg name="uav_frame_name"      default="base_link"/>
    <arg name="hokuyo_frame_name"   default="hokuyo"/>
    
    <!-- #######################  Mavros   ######################################## -->
    <include file="$(find mavros)/launch/px4.launch">
        <arg name="fcu_url" value="/dev/ttyUSB0:921600" />
    </include>

    <rosparam file="$(find architecture)/cfg/realUav_params.yaml" command="load" />

    <!-- #######################  Octomap  ######################################## -->
    

    <!-- #######################  Architecture nodes  ############################# -->
    <node name="mav_comms"          type="mav_comms_node"       pkg="architecture"  output="screen" />

    <!-- #######################    TF    ############################# -->    
    <!-- <node pkg="tf" type="static_transform_publisher" name="local_position_to_hokuyo" args="0.22 0 -0.2 0 -0.872665 3.14159 base_link laser 100" /> -->
    <node pkg="tf" type="static_transform_publisher" name="world_to_map" args="0 0 0 0 0 0 1 world map 100" />


    <!-- #######################  Hokuyo  ############################# -->
    
</launch>

